docs/content/preview/yugabyte-platform/prepare/networking-kubernetes.md

```diff
@@ -0,0 +1,75 @@
+---
+title: YugabyteDB Anywhere networking requirements for Kubernetes
+headerTitle: Networking requirements for Kubernetes
+linkTitle: Kubernetes
+description: Added requirements for deploying YugabyteDB Anywhere on Kubernetes.
+headContent: Added requirements for YugabyteDB Anywhere on Kubernetes
+menu:
+  preview_yugabyte-platform:
+    identifier: networking-kubernetes
+    parent: networking
+    weight: 10
+type: docs
+---
+
```

Comment:
Should we add a heading like "Basic requirements" here? Otherwise, it seems like the focus of this page is multi-cluster, which is not usually the common focus.